torchgeo/datasets/whispers23.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""WHISPERS23 dataset."""
+
+import glob
+import os
+from typing import Callable, Optional
+
+import numpy as np
+import rasterio
+import torch
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, extract_archive
+
+
+class WHISPERS23(NonGeoDataset):
+    """WHISPERS23 dataset.
+
+    The `WHISPERS23 <https://www.ieee-whispers.com/cross-city-challenge/>`__
+    dataset or the Cross-City Multimodal Semantic Segmentation Challenge
+    dataset is a dataset for land cover land use (LULC) semantic segmentation.
```

Comment:
Is there information how to download this dataset or is it a closed competition and this is just providing data loading for that?